packages/volto-razzle/package.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "@volto/razzle",
+  "description": "A Volto fork of Razzle - Create server-rendered universal JavaScript applications with no configuration",
+  "license": "MIT",
+  "author": "Jared Palmer <jared@palmer.net>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/plone/volto.git",
+    "directory": "packages/volto-razzle"
+  },
+  "version": "1.0.0-alpha.0",
+  "bin": {
+    "razzle": "./bin/razzle.js"
+  },
+  "files": [
+    "babel.js",
+    "bin",
+    "config",
+    "scripts",
+    "template"
+  ],
+  "dependencies": {
+    "@babel/plugin-transform-modules-commonjs": "^7.12.1",
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.4.3",
+    "autoprefixer": "^10.2.3",
+    "babel-jest": "^26.3.0",
+    "babel-loader": "^8.0.6",
+    "babel-plugin-transform-define": "^2.0.0",
+    "buffer": "^6.0.3",
+    "chalk": "^4.1.0",
+    "clean-css": "^5.0.1",
+    "copy-webpack-plugin": "^6.1.1",
+    "css-loader": "^5.0.0",
+    "css-minimizer-webpack-plugin": "^1.2.0",
+    "deepmerge": "^4.2.2",
+    "dotenv": "^8.2.0",
+    "dotenv-expand": "^5.1.0",
+    "file-loader": "^4.3.0",
+    "fs-extra": "^9.1.0",
+    "inquirer": "^7.3.3",
+    "jest": "^26.4.2",
+    "mri": "^1.1.4",
+    "null-loader": "^4.0.1",
+    "pnp-webpack-plugin": "^1.6.4",
+    "postcss": "^8.2.4",
+    "postcss-load-config": "^3.0.0",
+    "postcss-loader": "^4.2.0",
+    "process": "^0.11.10",
+    "razzle-start-server-webpack-plugin": "4.2.18",
+    "react-dev-utils": "^11.0.4",
+    "react-refresh": "^0.9.0",
+    "release-it": "^19.0.4",
+    "resolve": "^1.17.0",
+    "sade": "^1.4.2",
+    "source-map-support": "^0.5.16",
+    "string-hash": "^1.1.3",
+    "style-loader": "^2.0.0",
+    "terminate": "^2.1.2",
+    "terser-webpack-plugin": "^2.3.5",
+    "tiny-async-pool": "^1.1.0",
+    "url-loader": "^2.3.0",
+    "webpack-manifest-plugin": "^3.1.1",
+    "webpackbar": "~4||~5"
+  },
+  "peerDependencies": {
+    "babel-preset-razzle": "4.2.18",
```

Comment:
@wesleybl did you investigate if we will need to fork this one too? As long as I remember, this one contained all the babel deps, which will be terrible outdated too.